sui_types/src/crypto.rs

```diff
@@ -393,6 +393,30 @@ impl AuthoritySignature {
     }
 }
 
+/// AuthoritySignInfoTrait is a trait used specifically for a few structs in messages.rs
+/// to template on whether the struct is signed by an authority. We want to limit how
+/// those structs can be instanted on, hence the sealed trait.
+/// TODO: We could also add the aggregated signature as another impl of the trait.
+///       This will make CertifiedTransaction also an instance of the same struct.
+pub trait AuthoritySignInfoTrait: private::SealedAuthoritySignInfoTrait {}
+
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct EmptyAuthoritySignInfo {}
```

Comment:
Why not call it `EmptySignInfo` since it is not tied to an authority and probably won't be used in an authority?